lib/hex_web/web/templates/package.html.eex

```diff
@@ -50,6 +50,11 @@ build_tools = @current_release && @current_release.meta["build_tools"] || []
           </ul>
         </dd>
       <% end %>
+
+      <%= if @hexdocs_url do %>
+        <dt>Hexdocs</dt>
```

Comment:
The conditional is fine, I think we need another place for the link on the page. I don't think "Documentation: Hexdocs" will look good.